spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+describe Post, type: :feature do
+  before :all do
+    @user1 = User.create(
+      name: 'John',
+      photo: 'https://images.unsplash.com/photo-1661956602116-aa6865609028?ixlib=rb-4.0.3&ixid=MnwxMjA3fDF8MHxlZGl0b3JpYWwtZmVlZHwxfHx8ZW58MHx8fHw%3D&auto=format&fit=crop&w=500&q=60',
+      bio: 'Here my story',
+      posts_counter: 3
+    )
+    @user2 = User.create(
+      name: 'meshu',
+      photo: 'https://images.unsplash.com/photo-1661956602116-aa6865609028?ixlib=rb-4.0.3&ixid=MnwxMjA3fDF8MHxlZGl0b3JpYWwtZmVlZHwxfHx8ZW58MHx8fHw%3D&auto=format&fit=crop&w=500&q=60',
+      bio: 'Here my story',
+      posts_counter: 0
+    )
+
+    @post1 = Post.create(title: 'First Post', text: 'Here is my post', comment_counter: 0, like_counter: 0,
+                         author: @user1)
+
+    @comment = Comment.create(text: 'Here is my comment', author: @user2, post: @post1)
+    puts @comment, @post1
+  end
+
+  # test if the user of the post has an image
+
+  it 'should  have an image' do
+    visit("users/#{@user1.id}/posts")
+    expect(page).to have_css('img', count: 1)
+  end
+
+  it 'should have a username' do
+    visit("users/#{@user1.id}/posts")
+
+    expect(page).to have_content @post1.author.name
+  end
+
+  # it 'should have the number of posts' do
+  #   visit("users/#{@user1.id}/posts")
+
+  #   expect(page).to have_content "Number of post: #{@post1.author.posts_counter.to_s}"
+  # end
```

Comment:
- [ ]  According to the project requirements 👇🏻 You should have Tests that assert you can see the number of posts the user has written. Kindly add this 👍🏻 

![Screenshot (1)](https://user-images.githubusercontent.com/79058364/208742598-79005e88-3ac1-4c12-a262-559dea2729bd.png)

- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much more noisy, It is always a good practice to keep your code clean from such comments. 😉 👍🏻